constellations/gemini-user/src/afh/afh.js

```diff
@@ -0,0 +1,152 @@
+                            row={row}
+                            onChangeCheckbox={this.onChangeCheckbox}
+                        />
+                    </div>
+
+                    <div className="session-details">
+                        {sessionDate} {row.timeString} <br />
+                        {row.title} {row.notes} <br /> {row.locationId}
+                    </div>
+                </div>
+            );
+        });
+
+        return (
+            <div id="view-afh">
+                <h1>Ask for Help</h1>
+                <div className="description">
+                    We provide free sessions for students to ask for additional
+                    assistance on any of our program subjects. Please fill the
+                    form to let us know you are coming. You must be registered
+                    with one of our programs to attend.
+                </div>
+
+                <h1>Ask for Help Sessions by Subject</h1>
+                <div className="tab">
+                    <button
+                        className={
+                            this.state.currentTab == "Math" ? "active" : ""
+                        }
+                        onClick={() => this.openSubject("Math")}>
+                        Math
+                    </button>
+                    <button
+                        className={
+                            this.state.currentTab == "English" ? "active" : ""
+                        }
+                        onClick={() => this.openSubject("English")}>
+                        English
+                    </button>
+                    <button
+                        className={
+                            this.state.currentTab == "Computer Programming"
+                                ? "active"
+                                : ""
+                        }
+                        onClick={() =>
+                            this.openSubject("Computer Programming")
+                        }>
+                        Computer Programming
+                    </button>
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "Math" ? "showTab" : "hide"
+                    }>
+                    {showSessions}
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "English" ? "showTab" : "hide"
+                    }>
+                    {showSessions}
+                </div>
+
+                <div
+                    className={
+                        this.state.currentTab == "Computer Programming"
+                            ? "showTab"
+                            : "hide"
+                    }>
+                    {showSessions}
+                </div>
+            </div>
+        );
+    }
+}
+
+class CheckboxInput extends React.Component {
+    state = {
+        isActive: "false",
```

Comment:
make these boolean values so `isActive: false`.

And similar below, you can compare values by using
`this.state.isActive == false` Or just simply `!this.state.isActive`